rst2pdf/tests/conftest.py

```diff
@@ -309,27 +323,29 @@ def _build(self):
                 stderr=subprocess.STDOUT,
             )
             retcode = 0
+
+            output_file = os.path.join(OUTPUT_DIR, self.name + '.pdf')
+            no_pdf = os.path.exists(os.path.join(INPUT_DIR, self.name + '.nopdf'))
+            if not os.path.exists(output_file) and not no_pdf:
+                self._fail(
+                    'File %r was not generated'
+                    % (os.path.relpath(output_file, ROOT_DIR),),
+                    output,
+                )
+            elif os.path.exists(output_file) and no_pdf:
+                self._fail(
+                    'File %r was erroneously generated'
+                    % (os.path.relpath(output_file, ROOT_DIR),),
+                    output,
+                )
```

Comment:
Why the move? This will mean we won't get a log if either of these conditions occurs

Reply:
I missed what `.nopdf` does. I've put this code block back and documented `.nopdf`…